src/test/resources/files/bugs/713/testConstantArrayIndexIsset.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+define('FOO', []);
+
+if (isset(FOO['foo'])) {}
```

Comment:
Does it work for two, three or more levels, too?

```suggestion
if (isset(FOO['foo'])) {}

if (isset(FOO['foo']['bar'])) {}

if (isset(FOO['foo']['bar']['baz'])) {}
```

Reply:
Did not, fixed.